dozer/cogs/info.py

```diff
@@ -6,10 +6,21 @@
 datetime_format = '%Y-%m-%d %I:%M %p'
 
 class Info(Cog):
+	"""Commands for getting information about people and things on Discord."""
 	@guild_only()
 	@command(aliases=['user', 'userinfo', 'memberinfo'])
-	async def member(self, ctx, member : discord.Member = None):
-		"""Retrieve information about a member of the guild."""
+	async def member(self, ctx, member: discord.Member=None):
+		"""
+		Retrieve information about a member of the guild.
+		If no arguments are passed, information about the author is used.
+		**This command works without mentions.** Remove the '@' before your mention so you don't ping the person unnecessarily.
+		You can pick a member by:
+		- Username (`cooldude`)
+		- Username and discriminator (`cooldude#1234`)
+		- ID (`326749693969301506`)
+		- Nickname - must be exact and is case-sensitive (`"Mr. Cool Dude III | Team 1234"`)
```

Comment:
I might suggest using the `fuzzywuzzy` library like we did earlier for the feedback module, and finding the one with the greatest similarity. At the very least, keep it case insensitive.

Reply:
Nevermind, I just realized this is discord's built in user finder. Ignore me.